### PR TITLE
Adding size to Storage's NSDictionary representation

### PR DESCRIPTION
**Pull request by schmidt-sebastian**
Adding the missing size parameter to StorageMetadata.

As per Google internal issue: https://b.corp.google.com/issues/38041797

### CODE REVIEW

File: Example/Storage/Tests/Unit/FIRStorageMetadataTests.m [hunk 1]
```diff
@@ -43,8 +43,10 @@ - (void)testInitialzeFullMetadata {
     kFIRStorageMetadataMetageneration : @"67890",
     kFIRStorageMetadataName : @"path/to/object",
     kFIRStorageMetadataTimeCreated : @"1992-08-07T17:22:53.108Z",
-    kFIRStorageMetadataUpdated : @"2016-03-01T20:16:01.673Z"
+    kFIRStorageMetadataUpdated : @"2016-03-01T20:16:01.673Z",
+    kFIRStorageMetadataSize : @1337
   };
+
```
Comment (paulb777) [inline Example/Storage/Tests/Unit/FIRStorageMetadataTests.m]: delete spaces

File: Example/Storage/Tests/Unit/FIRStorageMetadataTests.m [hunk 2]
```diff
@@ -85,8 +89,10 @@ - (void)testDictionaryRepresentation {
...
```
Comment (paulb777) [inline Example/Storage/Tests/Unit/FIRStorageMetadataTests.m]: delete spaces

File: Firebase/Storage/FIRStorageMetadata.m [hunk 3]
```diff
@@ -180,6 +180,10 @@ - (NSDictionary *)dictionaryRepresentation {
   if (_path) {
     metadataDictionary[kFIRStorageMetadataName] = _path;
   }
+
```
Comment (paulb777) [inline Firebase/Storage/FIRStorageMetadata.m]: delete spaces

File: Firebase/Storage/FIRStorageMetadata.m [hunk 4]
```diff
@@ -180,6 +180,10 @@ - (NSDictionary *)dictionaryRepresentation {
   if (_path) {
     metadataDictionary[kFIRStorageMetadataName] = _path;
   }
+  
+  if (_size) {
+    metadataDictionary[kFIRStorageMetadataSize] = [NSNumber numberWithLong:_size];
```
Comment (paulb777) [inline Firebase/Storage/FIRStorageMetadata.m]: Fix warning

File: Firebase/Storage/FIRStorageMetadata.m (hunk 3 above)
Comment (schmidt-sebastian) [inline Firebase/Storage/FIRStorageMetadata.m]: This method has spaces for all of these blocks. I can remove them all if warranted.

File: Example/Storage/Tests/Unit/FIRStorageMetadataTests.m (hunk 2 above)
Comment (schmidt-sebastian) [inline Example/Storage/Tests/Unit/FIRStorageMetadataTests.m]: Done.

File: Example/Storage/Tests/Unit/FIRStorageMetadataTests.m (hunk 1 above)
Comment (schmidt-sebastian) [inline Example/Storage/Tests/Unit/FIRStorageMetadataTests.m]: Done.

File: Firebase/Storage/FIRStorageMetadata.m (hunk 4 above)
Comment (schmidt-sebastian) [inline Firebase/Storage/FIRStorageMetadata.m]: I addressed this warning and it no longer shows up in Travis. It now says that the Pod passed validation. I can't reproduce the original warning in XCode though (long is 64 bits on 64 bit machines, so it should have been sufficient).

File: Firebase/Storage/FIRStorageMetadata.m (hunk 3 above)
Comment (paulb777) [inline Firebase/Storage/FIRStorageMetadata.m]: travis is still complaining about one line in FIRStorageMetadata.m with trailing spaces.

You'll see the file and line number from the command line if you do git grep -n ' $'

Comment (schmidt-sebastian) [inline Firebase/Storage/FIRStorageMetadata.m]: Fixed

**Pull request merged**